package.json

```diff
@@ -28,14 +33,20 @@
     "storybook": "start-storybook",
     "pre-commit": "lint-staged",
     "build-storybook": "build-storybook -c .storybook -o .out",
-    "eslint": "eslint . --ext .tsx,.ts"
+    "eslint": "eslint . --ext .sass,.css"
```

Comment:
If you remove the .tsx,.ts then the eslint won't pick up lint errors in our typescript files

Reply:
I've added this back in.